### DISCUSSION TITLE
My Github account has been suspended.

### DISCUSSION THREAD

**Discussion by itzClay** [community/community#35583] [Other Feature Feedback, Questions, & Ideas] [answered]
Today my github account was suspended. I didnt receive any mails from the officials regarding the violations that I committed. Also i didnt receive any warnings. I had all of my work in my github account now due to the suspension all of those have been lost.

**Comment by airtower-luna** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Reply by ghost** [reply]
Same Here. They Shouldnt have done this without any warnings or Notice. And Now They are not even responding on the support.
